configure

```diff
@@ -323,10 +323,11 @@ valopt target "${CFG_HOST}" "GNUs ./configure syntax LLVM target triples"
 
 valopt localstatedir "/var/lib" "local state directory"
 valopt sysconfdir "/etc" "install system configuration files"
-valopt datadir "${CFG_PREFIX}/share" "install data"
-valopt infodir "${CFG_PREFIX}/share/info" "install additional info"
-valopt mandir "${CFG_PREFIX}/share/man" "install man pages in PATH"
-valopt libdir "${CFG_PREFIX}/lib" "install libraries"
+valopt datadir "share" "install data (relative to prefix)"
```

Comment:
In the compiler this defaults to `${CFG_PREFIX}/share`, so I'm curious how come this is just `share`? Presumably the two should be consistent, right?

(I'm not actually sure what the typical values for these arguments are)